slu/slu/utils/schema_parsing.py

```diff
@@ -0,0 +1,55 @@
+from typing import Any, Dict, List
+import pandas as pd
+from slu import constants as const
+from slu.utils.config import Config
+
+
+plugin_schema_parser_map = {
+    const.LIST_ENTITY_PLUGIN: list_entity_plugin_schema_parser
+}
+
```

Comment:
`plugin_schema_parser_map` seems to be used at only two places. where is it getting updated? is it always going to have only one key-value pair?

Reply:
The map will contain keys as plugin names and values as functions that produce plugin metadata.